app/src/components/MyPage.tsx

```
@@ -0,0 +1,172 @@
+                        </div>
+                        <h3 className="font-bold text-slate-800">Profile</h3>
+                    </div>
+
+                    <form onSubmit={handleUpdateProfile} className="space-y-4">
+                        <div className="space-y-1.5">
+                            <label className="text-[10px] font-bold text-slate-400 uppercase tracking-widest ml-1">Email</label>
+                            <input
+                                type="text"
+                                value={email}
+                                disabled
+                                className="w-full px-4 py-3 bg-slate-50 border border-slate-100 rounded-2xl text-slate-400 text-sm outline-none cursor-not-allowed"
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)